src/mpcCoreKit.ts

```diff
@@ -890,12 +932,9 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
    *
    * Exports the private key scalar for the current account index.
    *
-   * For keytype ed25519, consider using _UNSAFE_exportTssEd25519Seed.
+   * For signature type ed25519, consider using _UNSAFE_exportTssEd25519Seed.
    */
   public async _UNSAFE_exportTssKey(): Promise<string> {
-    if (this.keyType !== KeyType.secp256k1) {
```

Comment:
dont we need to add corresponding sigType check for this as per function comments on top?